test.js

```diff
@@ -280,32 +243,32 @@ describe('moment', function() {
   })
 
   describe('#iConvert', function() {
-    it('should convert 1999-04-01 to 1419-12-15', function() {
+    it('should convert 1999-04-01 to 1419-12-14', function() {
```

Comment:
This was correct date and shouldn't be changed `1999-04-01` is  `1419-12-15`. This means there is something wrong in #iConvert but the test should remain same.